src/components/react/IconList/index.tsx

```diff
@@ -1,51 +1,18 @@
 import * as icons from '@ubie/ubie-icons';
```

Comment:
If the order of the icons read in this line and the names of the icons generated by scripts/extractIconNames.js are not in the same order, the display will be corrupted.